apps/client/package.json

```diff
@@ -19,6 +19,7 @@
     "@tanstack/react-table": "^8.11.3",
     "autosize": "^6.0.1",
     "axios": "^1.2.0",
+    "chakra-react-select": "^4.7.6",
```

Comment:
I am not particularly confident on adding this as a dependency.

This seems to be maintained by a single person. From a quick review of the github repo it seems that the maintenance is not as good as we would like.

Chakra is about to release v3, we would be stuck waiting for this library to upgrade. We really are hoping that chakra will improve its performance or we would look to bail to another library.

Either way, we need to have independence on this task.

How about an array of checkboxes like we do in the cuesheet? not as nice, but simple enough?
We could look into improving later on

Reply:
sounds good